numpy/_core/fromnumeric.py

```diff
@@ -2469,8 +2469,8 @@ def any(a, axis=None, out=None, keepdims=np._NoValue, *, where=np._NoValue):
     >>> np.any([[True, False], [True, True]])
     True
 
-    >>> np.any([[True, False], [False, False]], axis=0)
-    array([ True, False])
+    >>> np.any([[True, False, True], [False, False, False]], axis=0)
```

Comment:
```suggestion
    >>> np.any([[True,  False, True ],
    ...         [False, False, False]], axis=0)
```
Aligning it like an array would print might be even clearer?  Just a suggestion, though.